packaging/package.yml/en.md

```diff
@@ -86,6 +87,20 @@ Step Name | Description
 **install** | This is where you should install the files into the final packaging directory, i.e. `make install`
 **check** | There is where tests / checking should occur, i.e. `make check`
 
+## Optimize values
+
+One or more optimize values can be specified in a list with the optimize key in the `package.yml` file. Several values can override or conflict with each other and should be used only where they provide a performance benefit, or fix a bug in the package or build.
+
+Step Name | Description
```

Comment:
Should probably be Optimize Value rather than a Step Name. Individuals might confuse it with actual dedicated steps like build, install, check, setup, etc.